### PR TITLE
chore: Add TF for wafv2 ipsets and regex sets

### PR DESCRIPTION
**Pull request by irmatov**

### CODE REVIEW

File: terraform/wafv2/prod/terraform.tf [hunk 1]
```diff
@@ -0,0 +1,7 @@
+terraform {
+#  backend "s3" {
```
Comment (roneli) [inline terraform/wafv2/prod/terraform.tf:2]: why is this commented out?

Comment (irmatov) [reply terraform/wafv2/prod/terraform.tf]: my bad. fixed.

**Review by spangenberg — APPROVED** [review]
LGTM

**Review by bbernays — CHANGES_REQUESTED** [review]

File: terraform/wafv2/prod/terraform.tf [hunk 2]
```diff
@@ -0,0 +1,7 @@
+terraform {
+  backend "s3" {
+    bucket         = "cq-provider-aws-tf"
+    key            = "backup"
```
[the remark this answers is not in the document]
Comment (irmatov) [reply terraform/wafv2/prod/terraform.tf:4]: :facepalm: Fixed, thanks!

**Review by bbernays — APPROVED** [review]

**Pull request merged**